packages/ember-cli-code-coverage/index.js

```diff
@@ -128,9 +128,9 @@ module.exports = {
    */
   _getIncludes() {
     return [
-      ...this._getIncludesForAppDirectory(),
-      ...this._getIncludesForAddonDirectory(),
       ...this._getIncludesForInRepoAddonDirectories(),
+      ...this._getIncludesForAddonDirectory(),
```

Comment:
The ordering between in-repo addons and addons is a bit tricky here. I don't think we can guarantee that in-repo addons are "clobbered" by "normal" addons (in fact I think I'd expect the opposite)... 🤔 

What do you think about using:

```suggestion
      ...this._getIncludesForAddonDirectory(),
      ...this._getIncludesForInRepoAddonDirectories(),
```

Reply:
That makes sense to me. I'd expect that the in-repo-addon would have a higher priority than an addon. I'll make the switch.